pkg/frontend/routine.go

```diff
@@ -120,6 +120,7 @@ func (routine *Routine) Loop(routineCtx context.Context) {
 		tenantCtx = context.WithValue(tenantCtx, defines.UserIDKey{}, tenant.GetUserID())
 		tenantCtx = context.WithValue(tenantCtx, defines.RoleIDKey{}, tenant.GetDefaultRoleID())
 		ses.SetRequestContext(tenantCtx)
+		ses.SetFromRealUser(true)
```

Comment:
Should this flag be per session or per operation?

Reply:
I think the flag is per session. In the code, it is set per operation. It is also per session